src/app/modules/gb-data-selection-module/gb-data-selection.component.html

```diff
@@ -99,29 +99,29 @@
           </div>
         </div>
       </md2-accordion-header>
-      <gb-table></gb-table>
+      <gb-data-table></gb-data-table>
     </md2-accordion-tab>
 
     <!-- The 4th step -->
-    <md2-accordion-tab>
-      <md2-accordion-header>
-        <div class="gb-data-selection-accordion-header">
-          <div class="gb-data-selection-accordion-sub-header-1">
-            Step 4: Export the selected data
-          </div>
-          <div class="form-inline gb-data-selection-accordion-sub-header-2">
-            <div class="gb-data-selection-accordion-sub-header-3">
-              <span class="gb-data-selection-emphasis-text">{{subjectCount_2}} subjects</span>,
-              <span *ngIf="queryService.showObservationCounts"
-                    class="gb-data-selection-emphasis-text">
-                {{observationCount_2}} observations
-              </span>
-            </div>
-          </div>
-        </div>
-      </md2-accordion-header>
-      <gb-export></gb-export>
-    </md2-accordion-tab>
+    <!--<md2-accordion-tab>-->
+      <!--<md2-accordion-header>-->
+        <!--<div class="gb-data-selection-accordion-header">-->
+          <!--<div class="gb-data-selection-accordion-sub-header-1">-->
+            <!--Step 4: Export the selected data-->
+          <!--</div>-->
+          <!--<div class="form-inline gb-data-selection-accordion-sub-header-2">-->
+            <!--<div class="gb-data-selection-accordion-sub-header-3">-->
+              <!--<span class="gb-data-selection-emphasis-text">{{subjectCount_2}} subjects</span>,-->
+              <!--<span *ngIf="queryService.showObservationCounts"-->
+                    <!--class="gb-data-selection-emphasis-text">-->
+                <!--{{observationCount_2}} observations-->
+              <!--</span>-->
+            <!--</div>-->
+          <!--</div>-->
+        <!--</div>-->
+      <!--</md2-accordion-header>-->
+      <!--<gb-export></gb-export>-->
+    <!--</md2-accordion-tab>-->
```

Comment:
Why is the export step commented out?

Reply:
Ok, I see that export will be moved somewhere else and this part here will be removed, right?

Reply:
will remove this